application/helpers/expressions/em_manager_helper.php

```diff
@@ -9485,7 +9492,7 @@ public static function ShowSurveyLogicFile($sid, $gid = null, $qid = null, $LEMd
                     . "onclick='window.open(\"$editlink\",\"_blank\")'>"
                     . $rootVarName . "</span>";
             }
-            $questionRow .= "</b><br />[<a target='_blank' href='$editlink'>QID $qid</a>]<br/>$typedesc [$type] $errText $sWarningsText</td>"
+            $questionRow .= "</b><br />[<a target='_blank' href='$editlink'>QID $qid</a>]<br/>$typedesc [$type] $themeDesc $errText $sWarningsText</td>"
```

Comment:
Maybe time to use some `sprintf(QID $s)` and wrap each part on span or div … 

<div class="question-link">[<a target='_blank' href='$editlink'>sprintf(gT("QID $s",$qid</a>]</div><div class="type-description">$typedesc</div> etc …